app/models/ucas_match.rb

```diff
@@ -24,12 +34,12 @@ def action_needed?
 
```

Comment:
```suggestion
return false if resolved?
```

as it's still flagged as "Action needed" even though it's resolved and it's breaking the component

<img width="671" alt="Screenshot 2020-12-04 at 17 09 41" src="https://user-images.githubusercontent.com/38078064/101193523-9a5ef800-3654-11eb-904a-fa83df8150aa.png">

Otherwise, LGTM!

Reply:
@AgaDufrat ✔️